grpc/src/main/proto/language_id.proto

```diff
@@ -14,7 +14,22 @@ message LanguageIdResponse {
   string langId = 1;
 }
 
+message IdResult {
+  string id = 1;
+  double score = 2;
```

Comment:
This can be float.

Reply:
It is double in the original class. IdResult in the ***Impl.

Reply:
Yes but we do not need that precision. But ok, double is fine.